### PR TITLE
Patch pannels visualization changes.

### PR DESCRIPTION
**Pull request by andrzej-jankowski**
Changes list:
- remarks are now display on patch panel
- patch pannels are now clickable
- link to rack edit panel was added to info box

Patch Panel visualization:
dump1

Infobox for patch pannels:
dump2

Depends on: https://github.com/allegro/ralph_assets/pull/611

### CODE REVIEW

File: src/ralph/ui/static/ui/js/app/rack/directives.js [hunk 1]
```diff
@@ -37,7 +37,11 @@ angular
                 $scope.$on('change_active_item', function(event, item){
                     $scope.u_range = [];
                     if (typeof item !== 'undefined' && item !== null) {
-                        for (var i = item.position; i <= item.position+item.height-1; i++) {
+                        var itemHeight = 1;     // for accessories...
+                        if (typeof(item.height) !== 'undefined') {
+                            itemHeight = item.height;
+                        }
+                        for (var i = item.position; i <= item.position+itemHeight-1; i++) {
```
Comment (vi4m) [inline src/ralph/ui/static/ui/js/app/rack/directives.js]: It's weird. Are you sure you closed your bracket { ?

Comment (vi4m) [inline src/ralph/ui/static/ui/js/app/rack/directives.js]: Never mind - its' good. My fault.

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**